C++ API parity: Softsign

pbelevich opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #27309 C++ API parity: MultiheadAttention
* #27382 C++ API parity: Linear
* #27538 C++ API parity: Threshold
* #27537 C++ API parity: Tanhshrink
* #27536 C++ API parity: Tanh
* **#27535 C++ API parity: Softsign**

Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

pbelevich committed:
C++ API parity: Softsign

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"




[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"




[ghstack-poisoned]

yf225 reviewed (approved):
Thanks for the awesome work @pbelevich!

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]

pbelevich committed:
Update on "C++ API parity: Softsign"


Differential Revision: [D17835408](https://our.internmc.facebook.com/intern/diff/D17835408)

[ghstack-poisoned]
